src/Faker/Provider/nl_NL/Company.php

```diff
@@ -36,47 +36,51 @@ class Company extends \Faker\Provider\Company
-    'Video', 'Sieraden', 'Kook', 'Woon', 'Pizza', 'Mode', 'Haar', 'Kleding', 'Antiek', 'Interieur', 'Gadget',
-    'Foto', 'Computer', 'Witgoed', 'Bruingoed', 'Broeken', 'Pakken', 'Maatpak', 'Fietsen', 'Speelgoed',
-    'Barbecue', 'Sport', 'Fitness', 'Brillen', 'Bakkers', 'Drank', 'Zuivel', 'Pret', 'Vis', 'Wijn', 'Salade',
-    'Terras', 'Borrel', 'Dieren', 'Aquaria', 'Verf', 'Behang', 'Tegel', 'Badkamer', 'Decoratie');
+    protected static $product = array(
+        'Keuken', 'Media', 'Meubel', 'Sanitair', 'Elektronica', 'Schoenen',
+        'Zorg', 'Muziek', 'Audio', 'Televisie', 'Pasta', 'Lunch', 'Boeken', 'Cadeau', 'Kunst', 'Tuin', 'Klus',
+        'Video', 'Sieraden', 'Kook', 'Woon', 'Pizza', 'Mode', 'Haar', 'Kleding', 'Antiek', 'Interieur', 'Gadget',
+        'Foto', 'Computer', 'Witgoed', 'Bruingoed', 'Broeken', 'Pakken', 'Maatpak', 'Fietsen', 'Speelgoed',
+        'Barbecue', 'Sport', 'Fitness', 'Brillen', 'Bakkers', 'Drank', 'Zuivel', 'Pret', 'Vis', 'Wijn', 'Salade',
+        'Terras', 'Borrel', 'Dieren', 'Aquaria', 'Verf', 'Behang', 'Tegel', 'Badkamer', 'Decoratie'
+    );
 
-    protected static $type = array('Markt', 'Kampioen', 'Expert', 'Concurrent', 'Shop', 'Expert', 'Magazijn',
-    'Dump', 'Store', 'Studio', 'Boulevard', 'Fabriek', 'Groep', 'Huis', 'Salon', 'Vakhuis', 'Winkel', 'Gigant',
-    'Reus', 'Plaza', 'Park', 'Tuin');
+    protected static $type = array(
+        'Markt', 'Kampioen', 'Expert', 'Concurrent', 'Shop', 'Expert', 'Magazijn',
+        'Dump', 'Store', 'Studio', 'Boulevard', 'Fabriek', 'Groep', 'Huis', 'Salon', 'Vakhuis', 'Winkel', 'Gigant',
+        'Reus', 'Plaza', 'Park', 'Tuin'
+    );
 
-    protected static $store = array('Boekhandel', 'Super', 'Tabakzaak', 'Schoenmaker', 'Kaashandel', 'Slagerij',
-    'Smederij', 'Bakkerij', 'Bierbrouwer', 'Kapperszaak', 'Groenteboer', 'Bioboer', 'Fietsenmaker', 'Opticien',
-    'Café', 'Garage');
+    protected static $store = array(
+        'Boekhandel', 'Super', 'Tabakzaak', 'Schoenmaker', 'Kaashandel', 'Slagerij',
+        'Smederij', 'Bakkerij', 'Bierbrouwer', 'Kapperszaak', 'Groenteboer', 'Bioboer', 'Fietsenmaker', 'Opticien',
+        'Café', 'Garage'
+    );
 
 
     /**
      * @example 'Fietsenmaker Zijlemans'
+     *
+     * @return string
      */
     public function company()
     {
-        $lastName = $this->generator->lastName;
-
         $determinator = static::numberBetween(0, 2);
-        switch ($determinator) {
-            case 0:
-                $companyName = static::randomElement(static::$product) . ' ' . static::randomElement(static::$type);
-                break;
-            case 1:
-                $companyName = static::randomElement(static::$product) . strtolower(static::randomElement(static::$type));
-                break;
-            case 2:
-                $companyName = static::randomElement(static::$store) . ' ' . $lastName;
-                break;
+
+        if (0 === $determinator) {
```

Comment:
what's wrong with switch / case?

Reply:
It's probably a matter of personal taste - personally I avoid to use it because

- it usually requires a lot more code
- it uses weak comparisons

But if you prefer, I can revert this change!

Reply:
Yes, I personally have nothing against switch/case, so please revert just that part

Reply:
Adjusted!